.rhiza/completions/rhiza-completion.bash

```
@@ -0,0 +1,47 @@
+    # Looks for lines like: target: ## description
+    opts=$(make -qp 2>/dev/null | \
+           awk -F':' '/^[a-zA-Z0-9][^$#\/\t=]*:([^=]|$)/ {split($1,A,/ /);for(i in A)print A[i]}' | \
+           grep -v '^Makefile$' | \
+           sort -u)
+
+    # Add common make variables that can be overridden
+    local vars="DRY_RUN=1 BUMP=patch BUMP=minor BUMP=major ENV=dev ENV=staging ENV=prod"
+    opts="$opts $vars"
+
+    # Generate completions
+    COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)